examples/backups/nats/cleanup.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Cleanup: tear down everything provisioned by the demo so the cluster returns
+# to its previous state.
+set -uo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/00-helpers.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./00-helpers.sh was not specified as input (see shellcheck -x).

(SC1091)